core/controllers/reader.py

```diff
@@ -21,7 +21,7 @@
 import jinja2
 
 from core.controllers import base
-from core.domain.classifier import lda_string_classifier
+from core.domain.classifier import LDAStringClassifier
```

Comment:
Ah. I think in this PR you should actually update it to use the registry, since you've just introduced the registry. That's kind of the point of the PR, and is a good test of whether the whole thing works properly.